fixtures/ios_app_with_templates/Tuist/Templates/custom/Template.swift

```diff
@@ -0,0 +1,20 @@
+import ProjectDescription
+
+let nameAttribute: Template.Attribute = .required("name")
+let platformAttribute: Template.Attribute = .optional("platform", default: "ios")
+
+let testContents = """
+// this is test \(nameAttribute) content
+"""
+
+let template = Template(
+    description: "Custom template",
+    attributes: [
+        nameAttribute,
+        platformAttribute
+    ],
+    files: [
+        .string(path: "custom_dir/custom.swift", contents: testContents)
```

Comment:
What about supporting using the attributes from the paths too? Like this example below.

![carbon](https://user-images.githubusercontent.com/663605/76950864-fc32d280-690a-11ea-92c8-3dad20d1745c.png)

Reply:
That is supported, too, should I add it to the fixture?

Reply:
Amazing! Yeah, that'd be great. That way tests will fail in the future if someone breaks it. Well thought